web_list_html_widget/README.rst

```diff
@@ -0,0 +1,45 @@
+Widget Html rendering in list views
+===================================
+
+This module allows to use a ``widget="html"`` for fields in list(tree) views.
+
+*WARNING* The content of the field will be rendered without HTML escaping and without any other security measure.
```

Comment:
HTML fields support a `sanitize` attribute. You could recommend (or enforce?) to use only HTML fields with the `sanitize=True` attribute set.